carma_wm/src/CARMAWorldModel.cpp

```diff
@@ -222,7 +222,24 @@ TrackPos CARMAWorldModel::routeTrackPos(const lanelet::BasicPoint2d& point) cons
   return tp;
 }
 
-std::vector<lanelet::ConstLanelet> CARMAWorldModel::getLaneletsBetween(double start, double end) const
+class LaneletDowntrackPair
```

Comment:
Is this downtrack relative to the contained lanelet or to the route overall? Also probably doesn't matter for now, but since this looks like an internal data structure might be worth some consideration about hiding this in a private namespace or something.

Reply:
The downtrack used by this method is route downtrack which is why is can be used with maneuvers. I'll wrap the class definition in an anonymous namespace {}
It does seem like a name that others might use on accident.